community/ibm-mesos-dev/pv_mesos.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
```

Comment:
Remove this file; it is not used during chart deployment.  If it were used, creating PersistentVolumes as part of a helm release is a bad practice; using `hostPath` is even worse.